src/Options.php

```diff
@@ -660,14 +614,10 @@ private function configureOptions(OptionsResolver $resolver)
             'serialize_all_object' => false,
             'sample_rate' => 1,
             'mb_detect_order' => null,
-            'auto_log_stacks' => true,
+            'attach_stacktrace' => false,
             'context_lines' => 3,
             'enable_compression' => true,
-            'current_environment' => 'default',
-            'environments' => [],
-            'excluded_loggers' => [],
-            'excluded_exceptions' => [],
```

Comment:
Not that important after all but I'm going to ask anyway: is there any reason regarding these options being moved to the end of the list?